### ISSUE TITLE
Nedis plug can not turn. On off from homey

### ISSUE THREAD

**Issue by DanKaihoj** [athombv/com.tuya#79]
### Homey version

2023 12.0.0

### Tuya app version

1.1.0

### App diagnostics report ID

3a1e5b84-4b23-41c9-a12a-adc263ca8d19

### Description

I have some nedis plugs, i can see the power consumption in homey but not turn on off, neither by pressing the icon like i can for my nous plugs, or by using an action card.

I have another brand of plug where i Also can not do it, so i do not know if it do a mistanke om the setup. I have set it up in tuya app and there it works fine?

I do not know how to add the device as other as mentioned below, does there exist a guide for that? 

### Device specification

<details>

```json
  { "device": ..., "specifications": ...}
```

</details>

**Comment by bobvandevijver** [comment]
Thank your for your report!

To add a device as "other" you only need to add a new Tuya device in Homey and instead of socket you select other (which is they device with the Tuya label. However, I believe I am already seeing what the issue is based on your submitted diagnostics report (which only contains 2 devices by the way).

```
onTuyaStatus {"switch":true,"countdown_1":0,"online":true}
```

@JELoohuis Seems that certain switches only use `switch` instead of `switch_1`.

**Comment by bobvandevijver** [comment]
Your issue should be fixed when we release 1.2.0: note that you will need to re-add your socket at that time.
